tenant/store.go

```diff
@@ -38,7 +40,8 @@ func (tenantStore *tenantStore) Entities() []interface{} {
 }
 
 type Tenant struct {
-	Id       uint64 `sql:"AUTO_INCREMENT"`
+	ID       uint64 `sql:"AUTO_INCREMENT"`
+	UUID     string `sql:"not null;unique"`
```

Comment:
Yes, I think it's confusing if we have ID (which is unique) and UUID. And if it's just called UUID then it sounds like we are determining it. But it's the ID used in the external system. So the name should reflect this somehow.